queue/src/fiskaltrust.Middleware.Localization.QueueIT/JournalProcessorIT.cs

```diff
@@ -2,20 +2,11 @@
 using System.Collections.Generic;
 using fiskaltrust.ifPOS.v1;
 using fiskaltrust.Middleware.Contracts.Interfaces;
-using Microsoft.Extensions.Logging;
 
 namespace fiskaltrust.Middleware.Localization.QueueIT
 {
     public class JournalProcessorIT : IMarketSpecificJournalProcessor
     {
-        private readonly ILogger<JournalProcessorIT> _logger;
-
-        public JournalProcessorIT(
-            ILogger<JournalProcessorIT> logger)
-        {
-            _logger = logger;
-        }
-
         public IAsyncEnumerable<JournalResponse> ProcessAsync(JournalRequest request)
         {
             throw new NotImplementedException();
```

Comment:
Should we maybe not just throw here, but log something? Are there even exports in Italy we need to support?